packages/vue/src/tracing.ts

```diff
@@ -66,7 +66,9 @@ export const createTracingMixins = (options: TracingOptions): Mixins => {
       continue;
     }
 
-    for (const internalHook of internalHooks) {
+    internalHooks.forEach((internalHook, hookType) => {
```

Comment:
Why change to `forEach`?

Reply:
It is meant to retrieve `hookType`. Are you thinking checking `internalHook === internalHooks[1]` would suffice?

Reply:
I have discarded the change and updated to compare `internalHook` with `internalHooks` directly.